Dockerfile.xdmod-openstack

```diff
@@ -14,7 +14,7 @@ RUN apt-get update \
 COPY ./hypervisor_facts.py /app/xdmod-openstack-hypervisor
 COPY ./moc_openstack_api_reporting.py /app/xdmod-openstack-reporting
 COPY ./GetConfigFiles.py /app/xdmod-get-config-files
-COPY ./xdmod-run-ingestor.sh /app/xdmod-run-ingestor.sh
+COPY ./xdmod-get-config.sh  /app/xdmod-get-config.sh 
```

Comment:
This file is only copied over for the `xdmod-openstack` image. Is it not necessary for `moc-xdmod` image?

Reply:
Yes, I was thinking of removing it when I refactor the shredder